internal/sbi/consumer/bsf_service.go

```diff
@@ -31,6 +32,18 @@ func (s *nbsfService) getHTTPClient() *http.Client {
 	return s.httpClient
 }
 
+// isBSFEnabled checks if BSF integration is enabled in configuration
+func (s *nbsfService) isBSFEnabled() bool {
+	config := factory.PcfConfig
+
+	// Default to true if BSF config is not specified (backward compatibility)
+	if config.Configuration.Bsf == nil {
+		return true
```

Comment:
I think the most of users may not need to deploy a BSF.
Could you set the default value to false?

Thank you!